From 14b2ab0ee15794cc32b198860e2978c8f51e18c7 Mon Sep 17 00:00:00 2001
From: commit-queue <commit-queue@webkit.org>
Date: Tue, 1 Nov 2011 10:17:06 +0000
Subject: [PATCH] bad codegen, pointer diff in
 JSC::JSCallbackConstructor::JSCallbackConstructor
 https://bugs.webkit.org/show_bug.cgi?id=60951

Adjust symbols visibility for WebCore.

Patch by Zeno Albisser <zeno.albisser@nokia.com> on 2011-11-01
Reviewed by Simon Hausmann.

* WebCore.pro:

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@98947 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 Source/WebCore/WebCore.pro |    2 ++
 1 file changed, 2 insertions(+)

--- a/Source/WebCore/WebCore.pro
+++ b/Source/WebCore/WebCore.pro
@@ -7,6 +7,8 @@ include($$PWD/../WebKit.pri)
 include($$PWD/WebCore.pri)
 include($$PWD/../JavaScriptCore/JavaScriptCore.pri)
 
+contains(QT_CONFIG, reduce_exports):CONFIG += hide_symbols
+
 TEMPLATE = lib
 TARGET = $$WEBCORE_TARGET
 CONFIG += staticlib
